app/controllers/sitemap_controller.rb

```diff
@@ -0,0 +1,46 @@
+class SitemapController < ApplicationController
+  DEFAULT_LASTMOD = "2021-03-01".freeze
```

Comment:
Is there a particular rational behind this date, or is it just a case of it needs to be static and in the past? I guess it makes sense as a sort of 'when the website launched' date?

Reply:
Yeah, I'm planning to do a sweep of the content repo and add `priority` and `date` entries to the markdown, I just picked a past date here for the time being.